chord_metadata_service/chord/management/commands/decamelize.py

```diff
@@ -0,0 +1,26 @@
+import json
+from django.core.management.base import BaseCommand
+from humps import decamelize
+
+
+class Command(BaseCommand):
+
+    help = "Converts the keys of a json document from camelCase to snake_case"
+
+    def add_arguments(self, parser):
+        parser.add_argument("data", action="store", type=str, help="JSON data file or DRS URI to validate.")
+
+    def handle(self, *args, **options):
+        data = options["data"]
+
+        with open(data, "r") as json_file:
+            json_data = json.load(json_file)
+            json_file.close()
+
+        json_data = decamelize(json_data)
+
+        with open(data, "w") as json_file:
+            json.dump(json_data, json_file, ensure_ascii=False, indent=4)
+            json_file.close()
```

Comment:
in `with` blocks you do not need to close the file explicitly, as exiting the `with` does this for you